sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM rust:1.32 as base-build
+
+ENV TMP_BUILD_DIR=/tmp/build
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV RUST_VERSION=1.32.0
+ENV PATH="$CARGO_HOME/bin:$PATH"
+
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        curl \
+        make \
+        musl-tools \
+        libseccomp-dev \
+        btrfs-tools \
+        socat \
+        sudo \
+        iptables \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install the Rust toolchain
+#
+# Install 1.32.0 due to Firecracker crashing with 148 code
```

Comment:
You shouldn't need to install rust or rustup, given that you're using the `rust:1.32` base image.  It should already have rust 1.32 and the rustup tool installed.

Reply:
Ah cool, so apparently the `rust:1.32` image doesn't have a default tool chain specified which was resulting in an error. However, that can be fixed by adding the `rust default "$RUST_VERSION" as a `RUN` command.